src/test/java/seedu/address/logic/commands/AddCommandTest.java

```
@@ -201,6 +201,11 @@ public Person getSelectedPerson() {
         public void setSelectedPerson(Person person) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public int getCount() {
+            return 0;
```

@case141 I overlooked this in my initial code review. This should throw an error (see all the other methods in this class).

---

This was actually automatically added.